src/main/java/org/openstreetmap/atlas/checks/commands/MapRouletteUploadCommand.java

```diff
@@ -81,14 +88,68 @@ protected void execute(final CommandMap commandMap,
                 }
                 catch (final IOException error)
                 {
-                    logger.error(
-                            MessageFormat.format("Error while reading {0}: ", logFile.getName()),
-                            error);
+                    logger.error("Error while reading {}:", logFile, error);
                 }
-            }
+            });
         });
     }
 
+    /**
+     * An enum containing the different types of input files that we can handle.
+     */
+    private enum HandledFileType
+    {
+        LOG,
+        ZIPPED_LOG
+    }
+
+    /**
+     * Determine whether or not this file is something we can handle, and classify it accordingly.
+     * 
+     * @param logFile
+     *            any file
+     * @return if this file is something this command can handle, the appropriate HandledFileType
+     *         enum value; otherwise, an empty optional.
+     */
+    private Optional<HandledFileType> maybeGetHandledType(final File logFile)
```

Comment:
We should avoid words like `maybe` in function names 😆 . How about `getOutputFileType`?